docs/documentation.html

```diff
@@ -231,6 +231,13 @@ <h3 id="generating-a-parser-javascript-api">JavaScript API</h3>
   <code>"umd"</code>); valid only when <code>output</code> is set to
   <code>"source"</code> (default: <code>"bare"</code>).</dd>
 
+  <dt><code>grammarSource</code></dt>
+  <dd>any object that represent origin of the input grammar. CLI would set
+  in to the string with the path to the file; parser in the network application
```

Comment:
"parser in the netowork application could" -> "parsers in network applications might"